src/main/java/io/codelex/classesandobjects/practice/videostore/Video.java

```diff
@@ -0,0 +1,50 @@
+package io.codelex.classesandobjects.practice.videostore;
+
+public class Video {
+    protected String title;
+    protected boolean flag = false;
```

Comment:
protected nav nepieciešams rakstīt 
`protected boolean flag = boolean flag` 
to flag varētu pārdēvēt kaut kā `isInStore` or something, tikai flag nav ta baigi paskaidrojošs